client/src/pages/Forecast.js

```diff
@@ -1,7 +1,33 @@
 import React from "react";
+import styled from "@emotion/styled";
+import windturbine from "../assets/windpower.png";
+import Footer from "../components/Footer";
+import BulpLogo from "../components/BulpLogo";
+
+const BackgroundContainer = styled.main`
+  background-image: url(${windturbine});
+  background-size: cover;
+  height: 100vh;
+`;
+
+const ContentContainer = styled.div`
+  height: 100vh;
```

Comment:
Leon always tells me to be careful when using vh and vw. I would use it in the outer element, say the ```body`` or the ```BackgroundContainer```, but then I would go with %. Try 100% and check if it works (I am not sure, but might work)